src/elements/alt-style/AltStyleElementForm.tsx

```diff
@@ -1,34 +1,90 @@
+import styled from "@emotion/styled";
+import { neutral } from "@guardian/src-foundations";
 import React from "react";
 import { FieldLayoutVertical } from "../../editorial-source-components/FieldLayout";
 import { FieldWrapper } from "../../editorial-source-components/FieldWrapper";
+import { RepeaterFieldMapIDKey } from "../../plugin/helpers/constants";
 import { AltStyleElementWrapper } from "../../renderers/react/AltStyleElementWrapper";
 import { createReactElementSpec } from "../../renderers/react/createReactElementSpec";
+import { Body } from "../../renderers/react/ElementWrapper";
+import {
+  LeftRepeaterActionControls,
+  RightRepeaterActionControls,
+} from "../../renderers/react/WrapperControls";
 import { altStyleFields } from "./AltStyleElementSpec";
 
 export const AltStyleElementTestId = "AltStyleElement";
 
+const RepeaterBody = styled(Body)`
+  padding: 8px 8px 16px 8px;
+  &:not(:last-child) {
+    border-bottom: 1px dashed ${neutral[60]};
+  }
+  position: relative;
+`;
+
+const RepeaterChild = styled("div")`
+  width: 100%;
+  padding: 0 8px;
+`;
+
+const ChildNumber = styled("div")`
+  position: absolute;
+  top: 16px;
+  left: 8px;
+  color: ${neutral[46]};
+  font-family: "Guardian Agate Sans", sans-serif;
+  line-height: 1;
+  font-weight: 700;
+  font-size: 1rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 24px;
+  padding: 4px;
+  width: 24px;
+  border: 1px solid ${neutral[60]};
+`;
+
 export const altStyleElement = createReactElementSpec({
   fieldDescriptions: altStyleFields,
   component: ({ fields }) => (
     <FieldLayoutVertical
       data-cy={AltStyleElementTestId}
       useAlternateStyles={true}
     >
-      {fields.repeater.children.map((repeater, index) => (
-        <div key={index}>
-          <FieldWrapper
-            headingLabel="Key Takeaway Title"
-            field={repeater.title}
-            useAlternateStyles={true}
+      {fields.repeater.children.map((repeater, index, children) => (
+        <RepeaterBody key={repeater[RepeaterFieldMapIDKey]}>
+          <ChildNumber>{index + 1}</ChildNumber>
+          <LeftRepeaterActionControls
+            removeChildAt={() => fields.repeater.view.removeChildAt(index)}
+            numberOfChildNodes={children.length}
+            minChildren={1}
           />
-          <FieldWrapper
-            headingLabel="Key Takeaway Content"
-            field={repeater.content}
-            useAlternateStyles={true}
+          {/*Use field index as key to avoid React render conflicts*/}
```

Comment:
think this comment line needs to move up above 57 (or remove it since, `key` in react is a fairly well know/understood concept)

Reply:
yep, needs to move up, I thought I had but missed it, thanks for spotting.

the comment is about that bug we encountered at the beginning of this work (it used to use the map index rather than the field index which caused the render conflicts on certain movements). perhaps I could clarify it? it's not saying use keys in React, it's saying field index is the valid key not map / node index.